docs/data/joy/getting-started/templates/order-dashboard/components/OrderTable.tsx

```diff
@@ -42,7 +55,96 @@ const rows = [
       name: 'Steve Hampton',
       email: 'steve.hamp@email.com',
     },
-    subscription: 'Monthly',
+  },
+  {
+    id: 'INV-1232',
```

Comment:
The new IDs weren't unique, it created console error spam in the console, fixed in #40316.

Learning, please continuously check that new code is free of warnings from:

- https://pagespeed.web.dev/
- https://validator.w3.org/
- https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh
- console warnings

👌

Reply:
Thanks for these learning links; much appreciated! More than just pointing something as wrong, learning about how to prevent it in the future is crucial, particularly for us designers who sometimes miss these more "under the hood" dev things! 🤙 Thanks!